spec/ticTacToe_spec.rb

```diff
@@ -11,56 +12,43 @@
   expect(@board.board).to eq(%w[1 2 3 4 5 6 7 8 9])
   end
 
-end
+  it 'Mark the board in the middle with an X' do
+    @board.mark_board('5', 'X')
+    expect(@board.board[4]).to eq('X')
+  end
 
-describe 'Game' do
+end
 
+describe 'PlayerChoice' do
```

Comment:
You forgot to move `PlayerChoice` specs out of `ticTacToe_spec.rb`

Reply:
Sure did. I'll make sure all of the class specs are in the appropriate files.